games-simulation/lincity-ng/lincity-ng-2.9_beta.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils multiprocessing
+
+DESCRIPTION="City simulation game"
+HOMEPAGE="https://github.com/lincity-ng/lincity-ng"
+SRC_URI="https://github.com/lincity-ng/lincity-ng/archive/lincity-ng-${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BitstreamVera CC-BY-SA-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="virtual/opengl
+	sys-libs/zlib
+	dev-libs/libxml2
```

Comment:
(I think zlogene wants this slotted)

Reply:
2 is the only SLOT now but yeah, the fact that it's not 0 generally means you should add `:2` here.

Reply:
Thank you!